roles/common/tasks/main.yml

```diff
@@ -149,12 +149,32 @@
   set_fact: ursula_revision={{ ursula_git_describe.stdout }}
 
 # run this last so we only update if run was successful
-- name: drop an motd with ursula metadata
-  template: src=etc/update-motd.d/90-ursula-motd dest=/etc/update-motd.d/90-ursula-motd mode=0755
+- name: drop an motd with ursula metadata for ubuntu
+  template:
+    src: etc/update-motd.d/90-ursula-motd
+    dest: /etc/update-motd.d/90-ursula-motd 
+    mode: 0755
   when: os == 'ubuntu'
 
+- name: drop an motd with ursula metadat for rhel
+  template:
+    src: etc/update-motd.d/90-ursula-motd
+    dest: /etc/profile.d/90-ursula-motd
+    mode: 0755
+  when: os == 'rhel'
+
+- name: run the ursula-motd script from profile.d
+  template:
+    src: etc/profile.d/ursula-motd.sh
+    dest: /etc/profile.d/ursula-motd.sh
```

Comment:
do we really need this?  can't we just write the template to `/etc/profile.d/ursula-motd.sh` rather than having a wrapper script ?